srcpkgs/gnome-books/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'gnome-books'
 pkgname=gnome-books
-version=3.34.0
-revision=2
+version=40.0
+revision=1
 build_helper="gir"
 build_style=meson
 hostmakedepends="pkg-config vala glib-devel librsvg docbook-xsl libxslt
  gettext appstream-glib desktop-file-utils"
 makedepends="gjs-devel gtk+3-devel libglib-devel webkit2gtk-devel tracker-devel
```

Comment:
`tracker-devel` here is not needed, `gnome-books` has been ported to `tracker3`

Reply:
fixed